tests/api/test_addon_uploads.py

```diff
@@ -196,6 +302,51 @@ def test_upload_listed_extension(base_url, session_auth):
     api_helpers.verify_addon_response_details(payload, response, 'create')
 
 
+@pytest.mark.serial
+@pytest.mark.create_session('api_user')
+def test_upload_extension_with_duplicate_guid(base_url, session_auth):
+    # get the guid of the addon submitted previously
```

Comment:
Could you add a description here?

Reply:
@jrbenny35 I've added a description to the test - https://github.com/mozilla/addons-release-tests/pull/585/files#diff-7e09b646bd48b4a753d19587391a44a62c8ea49751a01a521194f4e3fea1da5eR307-R310
Hope this gives some clarity to the scope of this test. If it's still unclear, let me know and I'll try to think of something better :)